src/Build.OM.UnitTests/Definition/ProjectItem_Tests.cs

```diff
@@ -780,30 +780,30 @@ public void ProjectGetterResultsInDriveEnumerationException()
         /// on the command line for it to pass. Otherwise, the following test results in an
         /// UnauthorizedAccessException while attempting to access the root of the drive.
         /// </summary>
-        //  [Fact]
-        //  public void ProjectGetterResultsInDriveEnumerationWarning()
-        //  {
-        //      using (var env = TestEnvironment.Create())
-        //      {
-        //          ChangeWaves.ResetStateForTests();
-        //          env.SetEnvironmentVariable("MSBUILDFAILONDRIVEENUMERATINGWILDCARD", "0");
-        //          BuildEnvironmentHelper.ResetInstance_ForUnitTestsOnly();
+        //[Fact]
```

Comment:
**Recent change:**

1. I added more tests for logging a warning. As Nathan suggested, should all of these commented out tests be completely removed as no one might look at them? Or should I work on mocking a filesystem (which might be a bit complicated considering how many codepaths lead to FileMatcher)?